quizMySQL/GameActivity.cs

```diff
@@ -0,0 +1,252 @@
+﻿using Android.App;
+using Android.Content;
+using Android.Widget;
+using Android.OS;
+using System.IO;
+using System.Data;
+using quizMySQL;
+using MySql.Data.MySqlClient;
+using System;
+using System.Collections.Generic;
+using System;
+using quizMySQL.DataBase;
+using Newtonsoft.Json;
+
+
+namespace quizMySQL
+{
+    [Activity(Label = "GameActivity", MainLauncher = false, Icon = "@drawable/icon")]
+    public class GameActivity : Activity
+    {
+        QuizGame game;
+        TextView questionNoTxt;
+        TextView question;
+        Button buttonA;
+        Button buttonB;
+        Button buttonC;
+        Button buttonD;
+
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+            // Set our view from the "main" layout resource
+            SetContentView(Resource.Layout.Game);
+            game = new QuizGame();
+            game.database = new DbManager("Server=10.0.2.2;Database=quiz;Uid=root;Allow User Variables=True;");
+            game.populatingQuestionList(game.database);
+            assignViewToVar();
+            refreshQuestionPage();
+            buttonA.Click += delegate { btn1Click(); };
```

Comment:
can you add the handlers from the UI xml definition point of view? so you don't have to do this in code behind?